src/test/java/org/simplify4u/plugins/keyserver/PGPKeysServerClientIT.java

```diff
@@ -120,7 +118,7 @@ public void testClient(String keyServerUrl) throws Exception {
 
         tempFile.deleteOnExit();
 
-        final PGPKeysServerClient client = PGPKeysServerClient.getClient(keyServerUrl);
+        final PGPKeysServerClient client = PGPKeysServerClient.getClient(keyServerUrl, null);
```

Comment:
maybe it is possible be `MockServer` to test if connection is made using proxy?

Reply:
I think in that case you need a separate proxy or one running inside the test which sets a flag that the connection is done using a proxy. On server side using HTTP it is hard to detect or even impossible if a HTTP proxy was in between  (usually only by fingerprinting the TCP connection like buffer sizes etc.).